frontend/i18n-scripts/README.md

```diff
@@ -0,0 +1,56 @@
+# i18n-scripts
+
+The i18n-scripts folder contains all homebrewed scripts for i18n workflow automation
+in OpenShift.
+
+## Languages
+The list of languages we're using are stored in `languages.sh`. All scripts
+that rely on language information import this variable, and it should be updated
+whenever OpenShift gains support for an additional language.
+
+## Memsource Automation
+We have created two scripts to handle pushing and pulling translations to/from
+Memsource, the tool the Red Hat Globalization team uses for translation jobs.
+
+Before running either tool, you must first install the [unofficial Memsource CLI client](https://github.com/unofficial-memsource/memsource-cli-client#pip-install).
+You also have to [configure it with your Memsource login info](https://github.com/unofficial-memsource/memsource-cli-client#configuration-red-hat-enterprise-linux-derivatives).
+
+On MacOS, you must also install rename, a GNU command line utility, before running the download script.
+* You can use `brew install rename` to install rename if you need it
```

Comment:
Does this work on Linux or Windows?  (do we care?)

Reply:
It should work on Linux! The Memsource CLI Client should work on RHEL (see the docs) and I believe Linux has rename included.

Reply:
I haven't tested it myself though.